featureflags/sse_client.py

```diff
@@ -109,7 +109,15 @@ def __next__(self):
                 self.buf += self.decoder.decode(next_chunk)
 
             except (StopIteration, requests.RequestException, EOFError) as e:
-                log.error(e)
+                if isinstance(e, StopIteration):
+                    log.error("Error when iterating through stream messages, attempting to resume")
```

Comment:
I say resume here as there is logic that will attempt to resume from the last message. For the other types, I say reconnect.